sdk/src/test/java/com/horizen/account/transaction/EthereumTransactionTest.java

```diff
@@ -44,7 +44,7 @@ public void BeforeEachTest() throws InvalidAlgorithmParameterException, NoSuchAl
     }
 
     @Test
-    public void ethereumRawTransactionTest() throws TransactionSemanticValidityException {
+    public void ethereumLegacyRawTransactionTest() throws TransactionSemanticValidityException {
```

Comment:
Add also separate serialization test class, like was done for `SidechainCoreTransactionSerializerTest`.
It may contain: serialization and regression tests for legacy and eip1559 txs. Both signed an non-signed versions.

Reply:
Yes, there is already a serializer test for signed and unsigned eth txs - just needs to be adjusted according to the latest changes